### PR TITLE
feat(cli): add command flags

### PR DESCRIPTION
**Pull request by veryCrunchy**
Adds 3 command flags to bypass [Inquirer.js](https://www.npmjs.com/package/inquirer)

Options:
  `-c, --create`
 `-m, --modify [presence]`
 `-t, --translate`

<details><summary>Option Flags Showcase</summary>
<p>

npx_pmd_-c
npx_pmd_-m_duolingo
npx_pmd_-m
npx_pmd_-t
npx_pmd_-m_notfound
</p>
</details>

tsconfig fix: Checks if the tsconfig.json file already exists
to prevent unnecessarily touching it, requiring the developer to restart the TS server

<details><summary>tsconfig fix</summary>
<p>
With fix:

npx_pmd_ts-fix
Without fix:

npx_pmd_ts-no-fix
</p>
</details>

### CODE REVIEW

File: packages/cli/src/actions/modify.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { createRequire } from "node:module";
 import { cp } from "node:fs/promises";
 import { fileURLToPath } from "node:url";
 
+import { Command } from "commander";
```
Comment (Timeraa) [inline packages/cli/src/actions/modify.ts:7]: Did you forget to add Commander to the package.json?

File: packages/cli/src/index.ts [hunk 2]
```diff
@@ -4,64 +4,76 @@ import "source-map-support/register.js";
 import chalk from "chalk";
 import { readFile } from "fs/promises";
 import inquirer from "inquirer";
+import { Command } from "commander";
```
Comment (Timeraa) [inline packages/cli/src/index.ts:7]: ditto

**Review by Timeraa — CHANGES_REQUESTED** [review]

File: packages/cli/src/actions/modify.ts (hunk 1 above)
Comment (veryCrunchy) [reply packages/cli/src/actions/modify.ts:7]: Mb..

**Comment by veryCrunchy** [comment]
Moved to #121

**Pull request closed without merging**